### PR TITLE
Add header options to portal-settings configlet

### PR DESCRIPTION
**Pull request by hvelarde**

### CODE REVIEW

File: src/brasil/gov/portal/controlpanel/portal.py [hunk 1]
```diff
@@ -1,27 +1,118 @@
...
+from zope import schema
 
 
-class ISettingsPortal(form.Schema):
+class ISettingsPortal(model.Schema):
     """Campos do formulário de configuração do Portal Padrão."""
-    esconde_autor = Bool(
+
+    # TODO: do we still need esconde_autor and esconde_data fields?
+    esconde_autor = schema.Bool(
```
Comment (rodfersou) [inline src/brasil/gov/portal/controlpanel/portal.py]: english, please

File: src/brasil/gov/portal/controlpanel/portal.py [hunk 2]
```diff
@@ -1,27 +1,118 @@
...
         title=_(u'Hides author'),
         description=_(u'Hide information about who created an item.'),
         default=False,
         required=False)
 
-    esconde_data = Bool(
+    esconde_data = schema.Bool(
```
Comment (rodfersou) [inline src/brasil/gov/portal/controlpanel/portal.py]: english, please

File: src/brasil/gov/portal/controlpanel/portal.py (hunk 1 above)
Comment (hvelarde) [reply src/brasil/gov/portal/controlpanel/portal.py]: that field is already there.

File: src/brasil/gov/portal/controlpanel/portal.py (hunk 2 above)
Comment (hvelarde) [reply src/brasil/gov/portal/controlpanel/portal.py]: that field is already there.

**Pull request merged**